README.md

```diff
@@ -27,13 +36,39 @@ compatibility. When possible, upgrade your Conjur version to match the
 when using integrations, choose the latest suite release that matches your Conjur version. For any 
 questions, please contact us on [Discourse](https://discuss.cyberarkcommons.org/c/conjur/5).
 
-### From PyPI
+### Supported Services
+
+- Conjur OSS v1.2.0+
```

Comment:
@sigalsax , check the Conjur OSS readmes for the exact suite version numbers
I believe the one we discussed is v1.5.0+suite1

https://docs.conjur.org/Latest/en/Content/ReleaseNotes/ConjurOSS-suite-RN.htm

Reply:
"+" -> and later